Fix build issue introduced in #297

**Describe the bug**
#297  introduced a build issue on Linux and Mac
```

clang++ -o build/linux_x86_64/clang_opt/usd-0.19.11_arnold-6.0.3.0/procedural/main.os -c -fPIC -Wno-deprecated-register -fno-operator-names -std=c++11 -w -O3 -fPIC -D_LINUX -DARNOLD_HAS_SCENE_FORMAT_API -I/home/S/buildbot/build_arnold-lnx/build/build/linux/clang_opt/dist/include -I/home/S/buildbot/build_arnold-lnx/build/build/linux/clang_opt/contrib/usd/19.11/include -I/home/S/buildbot/build_arnold-lnx/build/build/linux/clang_opt/contrib/usd/19.11/include/boost-1_61 -I/home/S/buildbot/build_arnold-lnx/build/virtualenv/bin -Ibuild/linux_x86_64/clang_opt/usd-0.19.11_arnold-6.0.3.0/procedural -Iprocedural -Itranslator/reader -Itranslator/writer procedural/main.cpp
procedural/main.cpp:318:42: error: cannot pass non-trivial object of type 'std::string' (aka 'basic_string<char>') to variadic function; expected type from format string was 'char *' [-Wnon-pod-varargs]
    AiMsgInfo("[usd] Saved scene as %s", filenameStr);
                                    ~~   ^~~~~~~~~~~
procedural/main.cpp:318:42: note: did you mean to call the c_str() method?
    AiMsgInfo("[usd] Saved scene as %s", filenameStr);
                                         ^
                                                    .c_str()
```

--- a/procedural/main.cpp
+++ b/procedural/main.cpp
@@ -315,7 +315,7 @@ scene_write
     writer->write(universe);       // convert this universe please
     stage->GetRootLayer()->Save(); // Ask USD to save out the file
     
-    AiMsgInfo("[usd] Saved scene as %s", filenameStr);
+    AiMsgInfo("[usd] Saved scene as %s", filenameStr.c_str());
     delete writer;
     return true;
 }

